aim/sdk/adapters/pytorch_lightning.py

```diff
@@ -117,15 +117,15 @@ def log_metrics(self, metrics: Dict[str, float],
             self.experiment.track(v, name=name, step=step, context=context)
 
     @rank_zero_only
-    def close(self) -> None:
-        super().close()
+    def finalize(self) -> None:
+        super().finalize()
         if self._run:
-            self._run.close()
+            self._run.finalize()
```

Comment:
this should stay as `self._run.close()` as the `finalize()` method will just index the results, rather than doing actual resource cleanup.